monthly-exams/exam-1/src/App.css

```diff
@@ -0,0 +1,32 @@
+.wrapper {
+  background-color: #f4f4f4;
+  height: 500px;
```

Comment:
We need the `wrapper` to cover the entire screen, we should use `height: 100vh;` and `width: 100vw;`